Makefile

```diff
@@ -1,12 +1,18 @@
 
-VERSION := $(shell cat cmd/synse.go | grep 'appVersion =' | awk '{print $$3}')
+VERSION := $(shell (git describe --tags --abbrev=0 --exact-match) || echo "no-tag")
```

Comment:
maybe instead of "no-tag", we could use something like "commit-<hash>"? not sure thats useful, but just a thought

Reply:
If I remove `--abrev=0` and `--exact-match` it will give that last tag on that branch followed by the current commit short-hash, something like:
`1.2.0-659es45`

Would you prefer that?